tunslip6.c

```diff
@@ -363,6 +375,110 @@ serial_to_tun(FILE *inslip, int outfd)
 unsigned char slip_buf[2000];
 int slip_end, slip_begin;
 
+int
+get_slipfd()
+{
+  const int start_i = 20;
+  int status, i, fd = -1;
+
+  for (i = start_i; i > 0; --i) {
+    if (i != start_i) {
+      ssystem("sleep 0.2s");
```

Comment:
I don't like this it feels hackish. Unfortunately we can't just usleep() because sigalarm is in use. Input on this would be nice.